students/Ajay_Randhawa/lesson03/list_lab.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python3
+def Series1():
```

Comment:
Function names should be snake_case in Python (as well as almost everything else, except Classes)